Program.cs

```diff
@@ -1,7 +1,14 @@
+using Microsoft.EntityFrameworkCore;
 using webapi.Services.UserService;
+using webapi.Data;
 
 var builder = WebApplication.CreateBuilder(args);
 
+var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
+
+// Configure the DataContext with the provided connection string
+builder.Services.AddDbContext<DataContext>(opt => opt.UseSqlServer(connectionString));
```

Comment:
need to look into concept on dependency injection again.

Reply:
Done